Sources/BijectiveDictionary/BijectiveDictionary+Initializers.swift

```diff
@@ -44,6 +44,47 @@ extension BijectiveDictionary {
         self._rtl = Dictionary(uniqueKeysWithValues: reversePairs)
     }
     
+    /// Creates a new dictionary from the left-right pairs in the given sequence, using a combining
+    /// to resolve conflicts.
+    /// - Parameters:
+    ///   - pairs: A sequence of left-right pairs to use for the new dictionary.
+    ///   - combine: A closure that is called when a conflict is encountered, receiving the left-right pair
+    ///     and a description of the conflict that exists.
+    /// - Note: If the combine closure does not resolve the conflict for a given left-right pair, then that left-right
```

Comment:
This is definitely something to consider.

I think the fundamental issue is that this initializer has to inform the user of 4 potential types of conflicts (as opposed to just conflicting keys with `Dictionary`) and allow them to be resolved. For this reason, I think the API will have to differ for this initializer, although, I am not yet sure if this is best approach.

I think it might be helpful to consider some of the ways in which this initializer will be used in real-life to ensure we take the right approach; I will try and come up with some examples.